tests/test-api.php

```diff
@@ -0,0 +1,392 @@
+<?php
+
+/**
+ * Test Co-Authors Plus' REST API
+ */
+global $wp_version;
+if (version_compare($wp_version, '4.4', '>=')) {
```

Comment:
Doing a big conditional around a class definition is a bit ugly. Is there a better way to achieve the goal?